main.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'app'
+
+def list_option
+  puts ''
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def handle_selection(option, app)
+  case option.to_i
+
+  when 3
+    create_person(app)
+
```

Comment:
When trying to create a person, I am getting the following error. Have a look and kindy fix this error before submitting for a re-review. ⬇️
![image](https://github.com/lorenzoworx/bookWarden/assets/33268927/d93da309-545e-4ab6-807f-6cf860ae2290)

Reply:
Please note that when a person student/teacher is created, I ran into the below error while trying to save both. Kindly fix this 👍🏽
![Screenshot from 2023-07-29 14-27-31](https://github.com/lorenzoworx/bookWarden/assets/44624138/7049fbce-d389-46c5-9ab7-c483652fd333)
![Screenshot from 2023-07-29 14-27-00](https://github.com/lorenzoworx/bookWarden/assets/44624138/d4101eb9-a206-4189-93ea-92ba8084ca40)